processor/src/main/java/com/joom/lightsaber/processor/analysis/Analyzer.kt

```diff
@@ -20,29 +20,35 @@ import com.joom.grip.Grip
 import com.joom.lightsaber.processor.ErrorReporter
 import com.joom.lightsaber.processor.model.InjectionContext
 import java.io.File
+import java.nio.file.Path
 
 class Analyzer(
   private val grip: Grip,
   private val errorReporter: ErrorReporter,
   private val projectName: String
 ) {
 
+  @Deprecated("Deprecated.", ReplaceWith("analyzePaths(files.map{ it.toPath() })"))
```

Comment:
I don't think we really need to preserve the legacy one. Let's just change its signature.